pkg/commands/workload_get.go

```diff
@@ -160,22 +160,52 @@ func (opts *WorkloadGetOptions) Exec(ctx context.Context, c *cli.Config) error {
 	// Print workload resources
 	c.Printf("\n")
 	if len(workload.Status.Resources) == 0 {
-		c.Infof("   Supply Chain resources not found.\n")
+		c.Infof(printer.AddPaddingStart("Supply Chain resources not found.\n"))
 	} else {
 		if err := printer.WorkloadResourcesPrinter(c.Stdout, workload); err != nil {
 			return err
 		}
 	}
 
+	// Deliverable
+	c.Printf("\n")
+	c.Boldf("Delivery\n")
+	// Print workload deliverable resources
+	wldDeliverable := getWorkloadResourceByKind(workload, cartov1alpha1.WorkloadDeliverableResourceKind)
+	var deliverableStatusReadyCond *metav1.Condition
+	notFoundMsg := printer.AddPaddingStart("Delivery resources not found.\n")
+	deliverable := &cartov1alpha1.Deliverable{}
+	if wldDeliverable == nil || wldDeliverable.StampedRef == nil {
+		c.Infof(notFoundMsg)
+	} else {
+		if err := c.Get(ctx, client.ObjectKey{Namespace: wldDeliverable.StampedRef.Namespace, Name: wldDeliverable.StampedRef.Name}, deliverable); err != nil {
```

Comment:
if Get fails with an error, `deliverable` is nil and since code path does not return here and in the section `deliverable.Status` is accessed so this could end up with nil pointer exception.

Return the error here or check the object not nil before accessing it

Reply:
Null object checked